tests/registry-command/test-ecr-user.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+set -ex
+
+# WARNING -- RACE-CONDITION: this test is not thread-safe (since it makes use of a shared user's secrets)
+# the lock.sh and unlock.sh scripts must first be run
+
+clearusersecrets() {
+    earthly secrets ls /user/std/ | xargs -r -n 1 earthly secrets rm
+}
+
+echo "here we go in test-ecr-user.sh"
```

Comment:
Same comment here about either using `$0` or just not including the name of the file.

Reply:
I was supposed to remove this before submitting this PR; thanks for catching this :)